documentation/modules/selecting-migration-network-for-virt-provider.adoc

```diff
@@ -15,6 +15,10 @@ If you do not select a migration network, the default migration network is the `
 You can override the default migration network of the provider by selecting a different network when you create a migration plan.
 ====
 
+include::snip-mtu-value.adoc[]
+
+// Is this still accurate?
+
```

Comment:
@sgratch Although it's not part of the Jira -- could you also check this procedure? I could not get hold of a QE environment.

Reply:
This is still feasible but instead of setting it via the menu next to the provider, it's done via the plan.

Go to plans-> plan details tab -> settings section. And edit the  `Transfer Network`  field:
![Screenshot from 2024-10-29 23-49-22](https://github.com/user-attachments/assets/02bdd9fa-2261-473b-80d8-fe9be140c95f)

Click edit on the pencil next to this field for openning the popup:
![Screenshot from 2024-10-29 23-50-01](https://github.com/user-attachments/assets/c2f5ab0a-cc66-4b6f-8827-1c73db8b338d)